### ISSUE TITLE
Polluting $PATH with setup & console

### ISSUE THREAD

**Issue by jdelStrother** [titusfortner/webdrivers#174]
#### Summary
The latest release adds `setup` & `console` as 'global' executables

#### Debug Info
Please provide the following information for bug reports:

* Webdrivers version: 4.4.0
* Ruby version: 2.7

#### Expected Behavior
`gem install webdrivers` doesn't add generically named executables like `setup` or `console` to the path

#### Actual Behavior
```
root@9ed62ec6e694:/# gem install webdrivers
Fetching nokogiri-1.10.9.gem
Fetching webdrivers-4.4.0.gem
Fetching mini_portile2-2.4.0.gem
Fetching rubyzip-2.3.0.gem
Fetching selenium-webdriver-3.142.7.gem
Fetching childprocess-3.0.0.gem
Successfully installed mini_portile2-2.4.0
Building native extensions. This could take a while...
Successfully installed nokogiri-1.10.9
Successfully installed rubyzip-2.3.0
Successfully installed childprocess-3.0.0
Successfully installed selenium-webdriver-3.142.7
Successfully installed webdrivers-4.4.0
6 gems installed
root@9ed62ec6e694:/# which console
/usr/local/bundle/bin/console
root@9ed62ec6e694:/# console
irb(main):001:0>
````

I believe this could be fixed by either setting `s.executables = []` in gemspec, or by giving the executables more explicit names (eg `webdrivers-console`)

**Comment by jpgeek** [comment]
Just hit this issue.  Conflicting with another gem ( omniauth-oauth2-generic) which also creates bin/console and setup.

**Comment by kapoorlakshya** [comment]
@jdelStrother Oops! Thanks for reporting this. Will fix it tonight.

**Comment by kapoorlakshya** [comment]
@jdelStrother @jpgeek Released fix in v4.4.1.

**Comment by jpgeek** [comment]
@kapoorlakshya Thanks!   You rock!
